spec.bs

```diff
@@ -171,15 +181,16 @@ It can be constructed using a string for each component, or from a shorthand str
 
 <xmp class="idl">
 typedef (USVString or URLPatternInit) URLPatternInput;
+typedef (URLPatternInput or URL) URLPatternMatchInput;
```

Comment:
You should not modify the actual IDL API; that's an undesired behavior change. You should only modify the algorithm that other specs call.

Reply:
ok.  I actually do not need them for the SW static routing API.  I have updated the original github issue and this PR's title and description to clarify that we focus on the fix for the SW static routing API.
And, I have reverted the change.